### PR TITLE
restore all url with pkg.go.dev for ACR readme

### PR DESCRIPTION
**Pull request by tadelesh**
restore all url with pkg.go.dev as the package has already been indexed

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Strange that it's been indexed before tagging.  The version is a hash, so I wonder if somebody grabbed it pre-release?

**Comment by tadelesh** [comment]
> Strange that it's been indexed before tagging. The version is a hash, so I wonder if somebody grabbed it pre-release?

I grabbed it to check the indexed content, also I want to add the pkg url in readme for the first preview.

File: sdk/containers/azcontainerregistry/README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ az acr create --name MyContainerRegistry --resource-group MyResourceGroup --loca
...
-This credential type works in both local development and production environments. 
+This document demonstrates using [azidentity.NewDefaultAzureCredential](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#NewDefaultAzureCredential) to authenticate.
+This credential type works in both local development and production environments.
 We recommend using a [managed identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) in production.
 
-Client and BlobClient accepts any [azidentity][https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity] credential. 
+[Client](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/containers/azcontainerregistry#Client) and [BlobClient](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/containers/azcontainerregistry#BlobClient) accepts any [azidentity][https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity] credential.
 See the [azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity) documentation for more information about other credential types.
```
Comment (xboxeer) [inline sdk/containers/azcontainerregistry/README.md:46]: can we paste env var that needed to make the default credential work quickly?

Comment (tadelesh) [reply sdk/containers/azcontainerregistry/README.md:46]: I don't prefer to do so. It is related with auth process that should be documented in `azidentity`.

Comment (chlowell) [reply sdk/containers/azcontainerregistry/README.md:46]: :+1: it's best to use `DefaultAzureCredential` as simply as possible in client library docs. We don't want to imply that environment variables are the only or the best way to configure authentication, or to describe the other options here.

**Review by xboxeer — APPROVED** [review]
LGTM

File: sdk/containers/azcontainerregistry/README.md (hunk 1 above)
Comment (xboxeer) [reply sdk/containers/azcontainerregistry/README.md:46]: Since this is the overall policy across data plane, I'm ok we have a link to azidentity

**Pull request merged**